_layouts/home.html

```diff
@@ -1,70 +1,295 @@
+              "Technical translation between researchers and RSEs": 1,
+              "Virtualization & Cloud computing": 4,
+              "Code optimization": 2
+            },
+            "Interpersonal": {
+              "Understanding of mental health, stress and burnout": 6,
+              "Building of community of practice": 3,
+              "Collaboration & building connections": 4,
+              "People management": 4,
+              "Leadership & communication skills": 1,
+              "Mentorship skills": 4,
+              "Stakeholder management": 4,
+              "Diversity, Equity and Inclusion": 4
+            },
+            "Leadership & management": {
+              "Project management": 7,
+              "Planning and organising events (workshops, meet-ups, conferences etc)": 1,
+              "Grant/proposal writing": 4,
+              "Evaluation and assessment": 1,
+              "Agile project management methodologies": 5,
+              "Recruitment": 4,
+              "Resource management (time, budget)": 7,
+              "Strategy": 4,
+              "Change management": 3,
+              "Domain expertise": 3,
+              "Research methodologies": 1,
+              "Ethics": 5,
+              "Domain-specific technical skills (AI/ML, research data and analytics, visualisation, computational science)": 2,
+              "Legal and regulatory": 7,
+              "Developing domain-specific training": 2
+            },
+            "Communication & dissemination": {
+              "Delivering training": 1,
+              "Writing and publication": 4,
+              "Public science communication": 5,
+              "Peer science communication": 1,
+              "Content and knowledge management": 4
+            }
+          };
+
+          const DATA_B = {
+            "Technical skills (general)": {
+              "Collaborative software development": 3,
+              "Coding conventions": 2,
+              "Software licensing": 1,
+              "Programming": 4,
+              "Environmental impact awareness": 3
+            },
+            "Technical (specific)": {
+              "Visualization": 2,
+              "Parallelization": 4,
+              "User Experience": 3,
+              "Technical translation between researchers and RSEs": 1,
+              "Code optimization": 2
+            }
+          };
+
+          const DATA_C = {
+            "Interpersonal": {
+              "Collaboration & building connections": 4,
+              "People management": 4,
+              "Stakeholder management": 4,
+              "Diversity, Equity and Inclusion": 4
+            },
+            "Leadership & management": {
+              "Project management": 7,
+              "Recruitment": 4,
+              "Resource management (time, budget)": 7,
+              "Strategy": 4,
+              "Domain expertise": 3,
+              "Research methodologies": 1,
+              "Developing domain-specific training": 2
```

Comment:
depending on what we're trying to illustrate with this chart, these could be separated into the "Domain expertise" category or just removed